apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -268,6 +267,19 @@
         public static String PATTERN = "%level %timestamp %thread %class : %msg %throwable";
     }
 
+    public static class StatusCheck {
+        /**
+         * Ignored exception list, also affect their subclasses.
+         */
+        public static String IGNORED_EXCEPTIONS = "";
+
+        /**
+         * Agent would do hierarchy check for the exception unless the max_recursive_depth equals to 0.And -1 means no limit.
```

Comment:
Don't provide no limitation. That could cause OOM

Reply:
All negative value should be considered as 0.

Reply:
ok